postgres/metadata.csv

```diff
@@ -30,32 +30,32 @@ postgresql.conflicts.lock,count,,query,,"Number of queries in this database that
+postgresql.autovacuumed,count,,,,Enabled with `relations`. The number of times this table has been vacuumed by the autovacuum daemon.,0,postgres,auto vac,
+postgresql.analyzed,count,,,,Enabled with `relations`. The number of times this table has been manually analyzed.,0,postgres,analyze,
+postgresql.autoanalyzed,count,,,,Enabled with `relations`. The number of times this table has been analyzed by the autovacuum daemon.,0,postgres,auto analyze,
 postgresql.last_vacuum_age,gauge,,second,,Last time at which this table was manually vacuumed (not counting VACUUM FULL).,0,postgres,age vacuum,
 postgresql.last_autovacuum_age,gauge,,second,,Last time at which this table was vacuumed by the autovacuum daemon.,0,postgres,age autovacuum,
 postgresql.last_analyze_age,gauge,,second,,Last time at which this table was manually analyzed.,0,postgres,age analyze,
 postgresql.last_autoanalyze_age,gauge,,second,,Last time at which this table was analyzed by the autovacuum daemon.,0,postgres,age autoanalyze,
-postgresql.index_rows_read,gauge,,row,second,The number of index entries returned by scans on this index.,0,postgres,idx rows read,
-postgresql.table_size,gauge,,byte,,"The disk space used by the specified table with TOAST data. Free space map and visibility map are not included.",0,postgres,tbl size,
+postgresql.index_rows_read,gauge,,row,second,Enabled with `relations`. The number of index entries returned by scans on this index.,0,postgres,idx rows read,
+postgresql.table_size,gauge,,byte,,Enabled with `relations`. "The disk space used by the specified table with TOAST data. Free space map and visibility map are not included.",0,postgres,tbl size,
 postgresql.relation_size,gauge,,byte,,"The disk space used by the specified table. TOAST data, indexes, free space map and visibility map are not included.",0,postgres,relation size,
-postgresql.index_size,gauge,,byte,,The total disk space used by indexes attached to the specified table.,0,postgres,idx size,
+postgresql.index_size,gauge,,byte,,Enabled with `relations`. The total disk space used by indexes attached to the specified table.,0,postgres,idx size,
 postgresql.toast_size,gauge,,byte,,The total disk space used by the toast table attached to the specified table.,0,postgres,toast size,
-postgresql.total_size,gauge,,byte,,"The total disk space used by the table, including indexes and TOAST data.",0,postgres,tot size,
+postgresql.total_size,gauge,,byte,,Enabled with `relations`. "The total disk space used by the table, including indexes and TOAST data.",0,postgres,tot size,
 postgresql.individual_index_size,gauge,,byte,,The disk space used by a specified index.,0,postgres,idx size,
-postgresql.table.count,gauge,,table,,The number of user tables in this database.,0,postgres,tbl count,
+postgresql.table.count,gauge,,table,,Enabled with `collect_count_metrics`. The number of user tables in this database.,0,postgres,tbl count,
```

Comment:
See Emma's comment here, this is enabled by default so I don't think this is necessary: https://github.com/DataDog/integrations-core/pull/16422/files#r1425983746